### PR TITLE
Release notes for 3.3.34-3 and 3.4.12-3

### PR DESCRIPTION
**Pull request by timtheisen**

### CODE REVIEW

File: docs/release/3.3/release-3-3-34-3.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+#### Enterprise Linux 7
+
+-   [igtf-ca-certs-1.91-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=igtf-ca-certs-1.91-1.osg33.el7)
+-   [osg-ca-certs-1.73-1.osg33.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=osg-ca-certs-1.73-1.osg33.el7)
+
+### RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    igtf-ca-certs osg-ca-certs osg-gums-config vo-client vo-client-edgmkgridmap vo-client-lcmaps-voms
```
Comment (brianhlin) [inline docs/release/3.3/release-3-3-34-3.md]: There aren't any `vo-client` changes, are there?

File: docs/release/3.4/release-3-4-12-3.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+#### Enterprise Linux 7
+
+-   [igtf-ca-certs-1.91-1.osg34.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=igtf-ca-certs-1.91-1.osg34.el7)
+-   [osg-ca-certs-1.73-1.osg34.el7](https://koji.chtc.wisc.edu/koji/search?match=glob&type=build&terms=osg-ca-certs-1.73-1.osg34.el7)
+
+### RPMs
+
+If you wish to manually update your system, you can run yum update against the following packages:
+
+    igtf-ca-certs osg-ca-certs osg-gums-config vo-client vo-client-edgmkgridmap vo-client-lcmaps-voms
```
Comment (brianhlin) [inline docs/release/3.4/release-3-4-12-3.md]: Same issue with the `vo-client` packages

Comment (timtheisen) [reply docs/release/3.4/release-3-4-12-3.md]: Good catch! Carl's script didn't function properly.

**Comment by timtheisen** [comment]
Hey Brian,
It should look good now.
...Tim

**Comment by timtheisen** [comment]
@brianhlin : I made the changes your requested.

**Pull request merged**